src/routes/partners/+page.svelte

```diff
@@ -0,0 +1,112 @@
+<script lang="ts">
+  import instagramGraphic from '$lib/assets/partners/instagram-graphic.png';
+  import logo from '$lib/assets/partners/blend-cards.png';
+  import party from '$lib/assets/partners/party.svg';
+  import piggy from '$lib/assets/partners/piggy.svg';
+  import formGraphic from '$lib/assets/organization/form-graphic.png';
+  import facebook from '$lib/assets/footer/facebook.png';
+  import instagram from '$lib/assets/footer/instagram.png';
+  import newsletter from '$lib/assets/footer/email_newsletter.svg';
+  import youtube from '$lib/assets/footer/youtube.png';
+</script>
+
+<div class="flex flex-wrap-reverse items-center justify-center gap-x-8 bg-[#f5f5ef] py-8 md:min-h-[calc(100vh-66px)]">
+  <img src={instagramGraphic} alt="Instagram Graphic" class="w-full object-contain md:w-[40%]" />
```

Comment:
I would change the alt to something a bit more descriptive like "A phone with an instagram video playing"

Also, let's set a min-h on this to avoid the layout shift as the image loads